src/like-service/app/Http/Controllers/LikeController.php

```diff
@@ -38,8 +40,11 @@ function doLike($request)
     function getLikeCountAndState($request)
     {
         $user_token = $request->cookie('jwt');
-        $postId = $request->header('postId');
+        $uri = $request->path();
```

Comment:
[NEED_INFO] Can not just use GET parameters here?

Reply:
We are. It's easiest to get to them through the URI